signer/src/schnorr.rs

```diff
@@ -58,8 +59,11 @@ impl<H: Hashable> Hashable for Message<H> {
 }
 
 impl<H: 'static + Hashable> Signer<H> for Schnorr<H> {
-    fn sign(&mut self, kp: &Keypair, input: &H) -> Signature {
-        let k: ScalarField = self.derive_nonce(kp, input);
+    fn sign(&mut self, kp: &Keypair, input: &H, packed: bool) -> Signature {
```

Comment:
Note: it should be able to reproduce the same behavior as [signFields](https://github.com/o1-labs/o1js/blob/main/src/mina-signer/mina-signer.ts#L148) for @dfstio. Note that "devnet" is hardcoded for the network.

As part of this patch, we need to ensure that the exposed Rust API allow the user to reproduce the same signature as the one in mina-signer.